libr/core/cmd_open.c

```diff
@@ -453,8 +454,8 @@ static void map_list(RIO *io, int mode, RPrint *print, int fd) {
 		default:
 			print->cb_printf ("%2d fd: %i +0x%08"PFMT64x" 0x%08"PFMT64x
 					" - 0x%08"PFMT64x" %s %s\n", map->id, map->fd,
-					map->delta, map->itv.addr, r_itv_end (map->itv) - 1,
-					r_str_rwx_i (map->perm), (map->name ? map->name : ""));
+					map->delta, r_io_map_get_from (map), r_io_map_get_to (map),
```

Comment:
can you do this chagne across the whole r2 codebase? this will make it more grep friendly

Reply:
$ git grep 'map->itv'|wc -l
     153 

this will make the PR quite big, so better do that in a separate PR. ignore this comment for now. would you like to do this change?

Reply:
I'll get this in the next PR. I will also include a new macro in r_io.h then, so we'll have:

```
#define r_io_map_get_from(map) map->itv.addr
#define r_io_map_get_to(map) ( r_itv_end (map->itv) - 1 )
#define r_io_map_get_end(map) r_itv_end (map->itv)
```

which should make things a lot cleaner since there's a lot of `map->itv.addr + map->itv.size`, `r_itv_end(map->itv)` and similar.